packages/reactivity/src/baseHandlers.ts

```diff
@@ -48,12 +48,12 @@ const arrayInstrumentations: Record<string, Function> = {}
 ;(['includes', 'indexOf', 'lastIndexOf'] as const).forEach(key => {
   const method = Array.prototype[key] as any
   arrayInstrumentations[key] = function(this: unknown[], ...args: unknown[]) {
-    const arr = toRaw(this)
+    const arr = toRaw(this) as any
```

Comment:
I'm not very familiar with TypeScript so if someone can suggest a more specific type, I'll gladly change it.

Reply:
I just tried it and it doesn't work. This is the Typescript error I get:
```
TS2684: The 'this' context of type '((...items: any[]) => number) | (() => any) | (() => any) | ((...items: any[]) => number) | { (start: number, deleteCount?: number | undefined): any[]; (start: number, deleteCount: number, ...items: any[]): any[]; }' is not assignable to method's 'this' of type '(this: unknown[]) => any'.   Type '{ (start: number, deleteCount?: number | undefined): any[]; (start: number, deleteCount: number, ...items: any[]): any[]; }' is not assignable to type '(this: unknown[]) => any'.
```